config/routes.rb

```diff
@@ -23,19 +25,21 @@
         end
       end
 
-        # 🔐 Auth
-        post "/login", to: "sessions#create"
-        get "/me", to: "sessions#show"
-
-        # 🛠 Product owner controls for managing tips
-        resources :reflection_tips, only: [ :create, :destroy, :update, :show ]
-
-        resources :orgs, only: [ :index ]
-        resources :memberships, only: [ :create ]
 
-        resources :keywords, only: [ :index ]
-        resources :insights, only: [ :index ]
-        resources :moneys, only: [ :index ]
+      # Org Owner Dashboard Controllers
+      resources :memberships, only: [ :create ]
+
+      # 🛠 Product Owner Dashboard Controllers
+      # resources :reflection_tips, only: [ :create, :destroy, :update ]
+      resources :dismissed_keywords, only: [:index, :destroy, :create]
+      resources :unmatched_keywords, only: [ :index ]
+  
+      resources :orgs, only: [ :index ]
+  
+  
+      resources :insights, only: [ :index ]
+  
+      resources :moneys, only: [ :index ]
```

Comment:
Adding comments to the routes.rb file provided visual grouping of related routes by their associated frontend dashboards. Clearly labeling which routes support specific dashboard views improves maintainability, supports faster onboarding, and ensures closer alignment between backend structure and frontend design decisions.